ui/dashboard.ts

```diff
@@ -0,0 +1,292 @@
+import {change_location, determine_location} from './navigation';
+import {set_ui_main_currency} from './topmenu';
+import {get_total_assets_value, iterate_saved_balances, total_table_add_balances, total_table_recreate} from './balances_table';
+import {assert_exchange_exists, format_currency_value, pages, settings} from './settings';
+import {setup_client_auditor, setup_log_watcher, showError} from './utils';
+import {create_or_reload_exchange} from './exchange';
+import {monitor_add_callback} from './monitor';
+import {prompt_sign_in} from './userunlock';
+import {ActionResult} from './model/action-result';
+import {ExchangeBalanceResult} from './model/exchange-balance-result';
+import {BlockchainBalances} from './model/blockchain-balances';
+import {ipcRenderer, remote} from 'electron';
+import {service} from './rotkehlchen_service';
+import {Currency} from './model/currency';
+
+function add_exchange_on_click() {
+    $('.panel a').click((event: JQuery.Event) => {
+        event.preventDefault();
+        const target = event.target as HTMLAnchorElement;
```

Comment:
So @kelsos this place here always returns undefined. Doing a print of the event itself I get the following:

```js
XX----> {
    "originalEvent": {
        "isTrusted": true
    },
    "type": "click",
    "target": {},
    "currentTarget": {
        "jQuery331031681506678370841": {
            "events": {
                "click": [
                    {
                        "type": "click",
                        "origType": "click",
                        "data": null,
                        "guid": 118,
                        "namespace": ""
                    }
                ]
            }
        }
    },
    "relatedTarget": null,
    "timeStamp": 17746.65,
    "jQuery33103168150667837084": true,
    "delegateTarget": {
        "jQuery331031681506678370841": {
            "events": {
                "click": [
                    {
                        "type": "click",
                        "origType": "click",
                        "data": null,
                        "guid": 118,
                        "namespace": ""
                    }
                ]
            }
        }
    },
    "handleObj": {
        "type": "click",
        "origType": "click",
        "data": null,
        "guid": 118,
        "namespace": ""
    },
    "data": null
}
```